web/help/GuideUnit1.md

```diff
@@ -593,21 +619,21 @@ more hints, expand the sections below for an example and explanation.
 
 As you continue with CodeWorld, you'll learn a few more functions.  For
 now, see if you can spruce up your nametag with some thicker lines.
-Try some solid shapes, too, and think about whether they are useful for
+Try some solid shapes too and think about whether they are useful for
 creating your nametag.
 
 Transformations
 ===============
 
 The nametags you built in the last section were pretty cool!  Still, they
 had some big limitations.  First, they were black and white.  Colors would
-make them much more exciting.  And second, all the shapes were in the
+make them much more exciting.  Second, all the shapes were in the
 center of the screen.  You can fix both of these problems using
 *transformations*.
 
 A **transformation** is a function that receives a picture called the
-preimage, and a description of something to change about it, and
-produces a new picture that is like the preimage except for that change.
+preimage, with a description of something to change about it, and then
```

Comment:
I used "and" deliberately there, because the preimage and the changes are both arguments side-by-side.  Students often fail to understand that the picture is an argument.  Maybe rewording would make this more clear.

For instance:

```
A **transformation** is a function whose arguments are:

1. a picture, called the preimage and
2. some changes to be made to the preimage.

The result of the transformation is a new picture that is like the preimage, except for that change.
```

Reply:
Not resolved.